models/ingredient/Ingredient.model.js

```diff
@@ -0,0 +1,15 @@
+import { Schema, model } from 'mongoose';
+
+const ingredientSchema = new Schema(
```

Comment:
how and when it should be added to DB?
does chef need to create ingredients before adding the dish?

Reply:
No. We'll adding it to db.

Reply:
1. how do we know about all the ingredients chef wants to use?
2. what if we won't have needed for chef ingredient? chef will DM us and ask to add it? :)

if we don't have answers for the questions above, think about providing chefs with field to provide their own ingredients list, saving it as a array of string in a dish document (`ingredients: ['salt', 'pepper', 'mustard']`) and removing this collection